src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -109,6 +113,11 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
             this.onDidChangeActiveTextEditor(this.docManager.activeTextEditor);
         }
         this.vscNotebook.onDidChangeNotebookEditorSelection(this.updateNativeNotebookContext, this, this.disposables);
+
+        const usingWebview =
+            !env.appName.includes('Insider') &&
+            !(await this.experimentService.inExperiment(Experiments.NativeNotebook));
```

Comment:
Please use `UseVSCodeNotebookEditorApi` intead of checking manually.
The logic keeps changing, hence its best to use the value `UseVSCodeNotebookEditorApi`.